IdentityServer/v7/Quickstarts/1_ClientCredentials/src/Client/Program.cs

```diff
@@ -11,7 +11,8 @@
 if (disco.IsError)
 {
     Console.WriteLine(disco.Error);
-    return;
+    Console.WriteLine(disco.Exception);
```

Comment:
When I first ran the client, it errored, and all I got was a generic error message with "see the inner exception for details".